### PR TITLE
added noiea target to goa rule. model_organism script fixed to includ…

### PR DESCRIPTION
**Pull request by dougli1sqrd**
…e goa taxa

### CODE REVIEW

File: pipeline/util/model_organism.py [hunk 1]
```diff
@@ -24,9 +24,8 @@ def read_taxa_for_path(dataset_path):
     with open(dataset_path) as group_file:
         group_data = yaml.load(group_file)
         gaf_data = [data for data in group_data["datasets"] if data["type"] == "gaf"]
-        taxa = [ taxon for gaf in gaf_data if gaf["taxa"] for taxon in gaf["taxa"] ]
+        taxa = [ taxon for gaf in gaf_data if gaf["taxa"] for taxon in gaf["taxa"] if taxon.split(":")[1] != "1" ]
```
Comment (kltm) [inline pipeline/util/model_organism.py]: I do not believe a special case is needed. But probably won't harm anything.

**Comment by kltm** [comment]
@dougli1sqrd Is `click` a standard library in py3?

**Comment by dougli1sqrd** [comment]
It's not, but I've been using it in our other pipeline tools, and you agreed at some point that click could be installed on our pipeline machines.

**Comment by kltm** [comment]
Looking at what we have, it's within the pipeline section that already has the requirements.txt run, so all good.

**Pull request merged**